docs/shared/recipes/nx-release/automatic-versioning-with-conventional-commits.md

```diff
@@ -0,0 +1,84 @@
+# Automatic Versioning with Conventional Commits
+
+If you wish to bypass the versioning prompt, you can configure Nx Release to defer to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard to determine the version bump automatically. This is useful for automating the versioning process in a CI/CD pipeline.
+
+## Enabling Automatic Versioning
+
+To enable automatic versioning via conventional commits, set the `release.version.conventionalCommits` property to `true` in `nx.json`:
+
+```json nx.json
+{
+  "release": {
+    "version": {
+      "conventionalCommits": true
+    }
+  }
+}
+```
+
+## Determining the Version Bump
+
+Nx Release will use the commit messages since the last release to determine the version bump. It will look at the type of each commit and determine the highest version bump from the following list:
+
+- 'feat' -> minor
+- 'fix' -> patch
+
+For example, if the git history looks like this:
+
+```
+  - fix(pkg-1): fix something
+  - feat(pkg-2): add a new feature
+  - chore(pkg-3): update docs
+  - chore(release): 1.0.0
+```
+
+then Nx Release will select the `minor` and elect to release version 1.1.0. This is because there is a `feat` commit since the last release of 1.0.0.
```

Comment:
```suggestion
then Nx Release will select the `minor` version bump and elect to release version 1.1.0. This is because there is a `feat` commit since the last release of 1.0.0.
```